pkg/kubelet/dockershim/docker_container.go

```diff
@@ -243,10 +239,12 @@ func (ds *dockerService) createContainerLogSymlink(containerID string) error {
 			glog.V(10).Infof("Docker logging driver is %s", dockerLoggingDriver)
 		}
 
-		if dockerLoggingDriver == dockerDefaultLoggingDriver {
-			glog.Errorf("Cannot create symbolic link because container log file doesn't exist!")
-		} else {
-			glog.V(5).Infof("Unsupported logging driver: %s", dockerLoggingDriver)
+		for _, driver := range criSupportedLogDrivers {
```

Comment:
Actually, there is already such a function, we should use it https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockershim/docker_service.go#L483-L494